CoinDock-api/database/migrations/2022_05_17_161749_create_coins_table.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('coins', function (Blueprint $table) {
+            $table->integerIncrements('id');
```

Comment:
use id() instead of integerIncrements('id')

Reply:
done